fips202/keccakf1600.h

```diff
@@ -16,7 +16,7 @@
 //
 // The struct is only exposed here to allow its construction on the stack.
 //
-typedef uint64_t keccakx4_state[KECCAK_WAY * KECCAK_LANES] ALIGN;
+typedef ALIGN uint64_t keccakx4_state[KECCAK_WAY * KECCAK_LANES];
```

Comment:
The movement of `ALIGN` should be explained in the commit message, and be done in a separate commit.

Reply:
Ah ok, the `;` separates things cleanly still. So it may well be that gcc/clang are agnostic to the placement.